data.sql

```diff
@@ -1,34 +1,66 @@
 /* Populate database with sample data. */
 
-INSERT INTO ANIMALS(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES(1, 'Agumon', '3-FEB-2020', 0, true, 10.23);
+INSERT INTO ANIMALS(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Agumon', '3-FEB-2020', 0, true, 10.23),
+  ('Gabumon', '15-NOV-2018', 2, true, 8),
+  ('Pikachu', '7-JAN-2021', 1, false, 15.04),
+  ('Devimon', '12-MAY-2017', 5, true, 11),
+  ('Charmander', '8-FEB-2020', 0, false, -11),
+  ('Plantmon', '15-NOV-2021', 2, true, -5.7),
+  ('Squirtle', '2-APR-1993', 3, false, -12.13),
+  ('Angemon', '12-JUN-2005', 1, true, -45),
+  ('Boarmon', '7-JUN-2005', 7, true, 20.4),
+  ('Blossom', '13-OCT-1998', 3, true, 17),
+  ('Ditto', '14-MAY-2022', 4, true, 22);
 
-INSERT INTO ANIMALS(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES(2, 'Gabumon', '15-NOV-2018', 2, true, 8);
 
-INSERT INTO ANIMALS(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES(3, 'Pikachu', '7-JAN-2021', 1, false, 15.04);
+BEGIN;
+INSERT INTO owners(full_name, age)
+VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+COMMIT;
```

Comment:
- [OPTIONAL] Please remove this keyword `COMMIT`. There should be no transaction here.
Don't forget to remove it in other areas as well.